velox/functions/prestosql/ArrayFunctions.h

```diff
@@ -316,6 +316,49 @@ struct ArraySumFunction {
   }
 };
 
+template <typename T>
+struct ArrayAverageFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE bool call(
+      double& out,
+      const arg_type<Array<double>>& array) {
+    // If the array is empty, then set result to null.
+    if (array.size() == 0) {
```

Comment:
I'm curious if array.empty() is available as well? It can make this code a bit more readable. CC: @laithsakka

Reply:
I can add it, we do no have it rn but its easy to add

Reply:
@laithsakka Thank you, Laith. That would be helpful.